contracts/adapters/AaveMigrationExtension.sol

```diff
@@ -575,4 +554,18 @@ contract AaveMigrationExtension is BaseExtension, FlashLoanSimpleReceiverBase, I
         });
         nonfungiblePositionManager.collect(params);
     }
+
+    function _checkAndRepaySubsidy(uint256 _maxSubsidy, uint256 _minWrappedSetTokenPosition) internal returns (uint256, uint256){
+        uint256 underlyingTokenBalance = underlyingToken.balanceOf(address(this));
+        uint256 subsidyAmount = _maxSubsidy.sub(underlyingTokenBalance); // Assumes 0 weth balance in this contract prior to tx
+        if (underlyingTokenBalance > 0) {
+            underlyingToken.transfer(msg.sender, underlyingTokenBalance);
+        }
+        uint256 wrappedSetTokenPositionAfter = uint256(setToken.getDefaultPositionRealUnit(address(wrappedSetToken)));
+        if(wrappedSetTokenPositionAfter< _minWrappedSetTokenPosition){
+            revert("MigrationExtension: Insufficient wrappedSetTokenPosition");
```

Comment:
this is already checked in the `TradeModule.trade()` call with  `_minReceiveQuantity`  https://github.com/IndexCoop/index-protocol/blob/9e6e7d865add19ebfa4735b9743ebeb3b5029533/contracts/protocol/modules/v1/TradeModule.sol#L122

Reply:
removed this check here [648211a](https://github.com/IndexCoop/index-coop-smart-contracts/pull/163/commits/648211a33a0d63b307e9eb88f81d2ec76c4fd503)

Reply:
Agree that it looks fine to remove this extra check.